### ISSUE TITLE
Maximum-scale on <meta> tag disables zooming on mobile devices

### ISSUE THREAD

**Issue by kimberlythegeek** [mozilla-services/screenshots#5018]
[detailed info on meta-viewport rule]( https://dequeuniversity.com/rules/axe/3.1/meta-viewport?application=axeAPI)
```
Rule Violated:
meta-viewport - Ensures <meta name="viewport"> does not disable text scaling and zooming
	URL: https://dequeuniversity.com/rules/axe/3.1/meta-viewport?application=axeAPI
	Impact Level: critical
	Tags: cat.sensory-and-visual-cues wcag2aa wcag144
	Elements Affected:
	1) Target: meta[name="viewport"]
		maximum-scale on <meta> tag disables zooming on mobile devices
```

**Comment by yehudab** [comment]
Ready to take this, if there are no objections.

**Comment by 6a68** [comment]
Sure. I've sent you an invite to be a contributor on this repo. Once you accept, I can formally assign it to you (but I don't think anyone else will open a PR in the meantime)

**Comment by yehudab** [comment]
Got the invite and accepted. Thanks!
